src/sentry/models/authidentity.py

```diff
@@ -53,20 +52,6 @@ def __str__(self):
     def get_audit_log_data(self):
         return {"user_id": self.user_id, "data": self.data}
 
-    # TODO(dcramer): we'd like to abstract this so there's a central Role object
-    # and it doesnt require two composite db objects to talk to each other
-    def is_valid(self, member):
```

Comment:
There is a call to this method in `sentry/models/project.py` on line 404, you'll likely have to clean that up too.

Reply:
Good call out